lib/sync_checker/formats/detailed_guide_check.rb

```diff
@@ -31,6 +31,7 @@ def expected_details_hash(edition)
           expected_details_hash.merge(
             national_applicability: edition.national_applicability
           ) if edition.nation_inapplicabilities.any?
+
```

Comment:
This commit doesn't do anything any more. I think either remove the commit entirely or rename it to "whitespace fix" or similar